eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/Retry.java

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs;
+
+import com.azure.core.amqp.exception.AmqpException;
+import com.azure.core.amqp.exception.ErrorCondition;
+import com.azure.eventhubs.implementation.ClientConstants;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public abstract class Retry {
+
+    public static final Retry NO_RETRY = new RetryExponential(Duration.ofSeconds(0), Duration.ofSeconds(0), 0);
+
+    private AtomicInteger retryCount = new AtomicInteger(0);
```

Comment:
`private final` rather than just `private`. I like to make it clear that this field variable should only be set once.

Reply:
You can just do `new AtomicInteger()`, it will set the initial count to 0.